### PR TITLE
Include tag in data feed listing

### PR DESCRIPTION
**Pull request by martinkersner**
# Description

* Include tag (`premium`, `standard` and `basic`) to low-level feed configuration
* removed `BUSD-USDT` aggregator and proxy deployment files
* unified `PAXG-USDT` and `PAX-USDT` to `PAXG-USDT`

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Summary by CodeRabbit

- **New Features**
  - Updated the system's data feed addresses to reflect the latest changes for various currency pairs and environments.
  - Introduced a new `tag` field in data feeds to categorize currency pairs more efficiently.
  - Enhanced the system to fetch feed information dynamically using an external API, improving data accuracy and relevance.

- **Chores**
  - Removed outdated addresses from the system to maintain up-to-date and streamlined operations.

- **Refactor**
  - Integrated the `axios` library for network requests, enhancing the efficiency of external data fetching.

### CODE REVIEW

File: contracts/v0.1/scripts/generate-contract-addresses.js [hunk 1]
```diff
@@ -1,10 +1,34 @@
 const path = require('path')
 const fs = require('fs')
+const axios = require('axios')
 const { loadJson, storeJson } = require('./utils.cjs')
 
 const ValidChains = ['baobab', 'cypress']
 const deploymentsPath = path.join(__dirname, '../deployments/')
 
+const getFeedTag = async (network, pairName) => {
+  url = `https://config.orakl.network/adapter/${network}/${pairName.toLowerCase()}.adapter.json`
```
Comment (nick-bisonai) [inline contracts/v0.1/scripts/generate-contract-addresses.js:10]: just optional comment, maybe you can use collected adapter json file and request url only once at the beginning

Comment (martinkersner) [reply contracts/v0.1/scripts/generate-contract-addresses.js:10]: I am not sure I understand. I think it should be called only once. It makes request only for `cypress` adapter.

Comment (nick-bisonai) [reply contracts/v0.1/scripts/generate-contract-addresses.js:10]: there's single json file collecting whole adapters together in following link

https://config.orakl.network/baobab_adapters.json

Comment (martinkersner) [reply contracts/v0.1/scripts/generate-contract-addresses.js:10]: Aha, now I get it! Good idea!

File: contracts/v0.1/scripts/generate-contract-addresses.js [hunk 2]
```diff
@@ -1,10 +1,34 @@
...
+  }
+
+  if (numFeeds > 8) {
+    return 'premium'
+  } else if (numFeeds > 5) {
+    return 'standard'
+  } else {
+    return 'basic'
+  }
+}
```
[the remark this answers is not in the document]
Comment (martinkersner) [reply contracts/v0.1/scripts/generate-contract-addresses.js]: fixed at https://github.com/Bisonai/orakl/pull/1364/commits/43e5f1f9e8f81f6a6cc1aae2284ac19d932896e1

File: contracts/v0.1/scripts/generate-contract-addresses.js (hunk 1 above)
Comment (martinkersner) [reply contracts/v0.1/scripts/generate-contract-addresses.js]: fixed at https://github.com/Bisonai/orakl/pull/1364/commits/0bfc2a56a467fe5fd652ee69e5711c7081016567

File: contracts/v0.1/scripts/generate-contract-addresses.js [hunk 3]
```diff
@@ -1,16 +1,45 @@
...
+          tag = 'standard'
+        } else {
+          tag = 'basic'
+        }
+        tags[feed.name] = tag
+      })
+    })
+
+  return tags
+}
```
[the remark this answers is not in the document]
Comment (martinkersner) [reply contracts/v0.1/scripts/generate-contract-addresses.js]: fixed at https://github.com/Bisonai/orakl/pull/1364/commits/13f8132c6c41acf08e63c60b0f279f903d944bdc

**Pull request merged**